packages/components/dropdown/index.ts

```diff
@@ -13,3 +13,5 @@ export const ElDropdownItem = withNoopInstall(DropdownItem)
 export const ElDropdownMenu = withNoopInstall(DropdownMenu)
 export * from './src/dropdown'
 export * from './src/tokens'
+
+export type DropdownInstance = InstanceType<typeof Dropdown>
```

Comment:
[It is recommended to refer to other code modifications](https://github.com/element-plus/element-plus/blob/dev/packages/components/backtop/src/instance.ts)

Reply:
Sorry about that. I followed https://github.com/element-plus/element-plus/blob/dev/packages/components/form/index.ts before. Updated.